man/docker-run.1.md

```diff
@@ -508,6 +509,18 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Docker to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+To control mount propagation property of volume one can use `:[r]shared`,
+`:[r]slave` or `:[r]private` propagation flag. Propagation property can
```

Comment:
'rshared' is recursive while 'shared' is not. Same is for other options. 'r' means recursive. So if source dir has submounts underneath it, then these submounts will be visible in the container too and recursive will make sure all the submounts undergo through propagation flag operation instead of just top level mount.